packages/styles/stories/Components/popover/popover.stories.js

```diff
@@ -34,176 +34,6 @@ export default {
     }
 };
 
-export const Alignment = () => `<div class="fddocs-container" style="margin-bottom: 200px">
```

Comment:
FYI, You can extract HTML files into the separate HTML files if you want to for proper HTML formating or something